main.py

```diff
@@ -14,6 +15,7 @@
 PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 os.chdir(PROJECT_ROOT)
 
+import windrecorder.flag_mark_note as flag_mark_note  # NOQA: E402
```

Comment:
use `from windrecorder import flag_mark_note`. Don't use `as` in such cases in the future.